src/wallet/rpcwallet.cpp

```diff
@@ -3017,6 +3022,7 @@ static UniValue fundrawtransaction(const JSONRPCRequest& request)
                             {"replaceable", RPCArg::Type::BOOL, /* opt */ true, /* default_val */ "", "Marks this transaction as BIP125 replaceable.\n"
                             "                              Allows this transaction to be replaced by a transaction with higher fees"},
                             {"conf_target", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "", "Confirmation target (in blocks)"},
+                            {"min_conf", RPCArg::Type::NUM, /* opt */ true, /* default_val */ 0, "The minimum confirmations to filter"},
```

Comment:
Whoopsie, default_value should be string. Should fix this by disallowing something else.